book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :book_rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @book_rentals = []
+  end
+
+  def new_rental(book)
+    @book_rentals.push(book)
+    book.book_rental = self
+  end
```

Comment:
- Hey, let's kindly make some improvements to your implementation by adding a date and person to the parameter and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 


```ruby
def new_rental(person, date)
    Rental.new(date, self, person)
  end
```

Reply:
Done!